clamnibs/base.py

```diff
@@ -189,5 +198,5 @@ def __init__(self, raw, apply_hil=True):
         self.participant = raw.participant
         self.session = raw.session
         self.forward_full = raw.forward_full
-        self.flip = raw.flip
+        if hasattr(raw, 'flip'): self.flip = raw.flip
```

Comment:
RawCLAM objects should always have a flip attribute (see logic above). If a flip is not present in .mat file, it is either computed in the data (in the case of no stim) or an error is thrown (in the case of stim).

Reply:
removed that part